packages/ui/src/components/vuestic-components/va-rating/VaRatingItem.demo.vue

```diff
@@ -1,77 +1,72 @@
 <template>
```

Comment:
Not a request from my side, just a piece of history:
I asked previous dev to do this demo just to help him streamline the component. :D
VaRatingItem is not meant as independent component. But I think we can keep  the demo given it has some value.